### DISCUSSION TITLE
Scroll feature

### DISCUSSION THREAD

**Discussion by cpyarger** [malcolmholmes/pico-clock-green-python#10] [Ideas]
I am thinking it would be useful to have an extension to the show_text function that takes any text that is too long for the display and scrolls through the characters.

**Comment by malcolmholmes** [comment]
That would be amazing!

I'm trying to see if I can get the display setting to happen on the second core, so that the main scheduler doesn't disrupt the updates of the display (which can I think cause it to dim or flicker slightly). But that shouldn't conflict with code to scroll characters. Fancy having a go at it?

**Comment by cpyarger** [comment]
If I can I will try to get around to it. I am poking the idea of making the serial api first. that way its easier to test
